task_processing/plugins/mesos/execution_framework.py

```diff
@@ -48,6 +50,9 @@
 
 class TaskMetadata(PRecord):
     agent_id = field(type=str, initial='')
+    pod_id = field(type=(str, type(None)), initial=None)
+    terminated = field(type=bool, initial=False)
+    terminal_raw_status_update = field(type=(type(None), Dict), initial=None)
```

Comment:
what's this for? I can't see it being read anywhere, only written to?

Reply:
Good catch! Removed it.